types/theo/index.d.ts

```diff
@@ -91,3 +86,19 @@ export interface FormatOptions {
         transformPropName?: (name: string) => string
     ) => void;
 }
+
+export function convert(options: ConvertOptions): Promise<string>;
+export function convertSync(options: ConvertOptions): string;
+export function registerFormat<T extends string = never>(
+    name: Format | T,
```

Comment:
Using generics will allow for arbitrary strings while still providing autocomplete for Theo's provided formats, transforms, and value transforms.